reana_cluster/configurations/reana-cluster-minikube-dev.yaml

```diff
@@ -1,6 +1,7 @@
 cluster:
   type: "kubernetes"
   version: "v1.14.0"
+  reana_url: "leticia-imac.dyndns.cern.ch"
```

Comment:
You can remove `reana_url` here and in the next file and we can use the [`--url`](https://reana-cluster.readthedocs.io/en/latest/cliapi.html#cmdoption-reana-cluster-u) option on the `reana-cluster` CLI when creating the cluster.